internal/helper/helper.go

```diff
@@ -454,3 +388,36 @@ func CalculateSHA256(fileName string) (string, error) {
 
 	return string(hasher.Sum(nil)), nil
 }
+
+// CheckTags iterates through the keys in a struct and looks for
+// a value passed in as a parameter. It returns the yaml name of
+// the key and an error. Currently only boolean values for the tags
+// are supported
+func CheckTags(searchStruct interface{}, tag string) (string, error) {
```

Comment:
Purely cosmetic question: any reason why `CheckTags()` returns a string (and error), and not just a boolean? From what I see from the current usage, we anyway simply just check if the return is `!= ""`, so it feels to me just working with true/false would make more sense? Since the return value is a key, so == to what is added as the argument, are there any other plans to use this helper differently?

Reply:
The string that is returned is used to create the error message when validating the YAML against the schema. It checks which field with `is_disk` is used and returns that, so the error can be more informative to the user.